fixtures/wrangler-dev-api-app/wrangler.toml

```diff
@@ -0,0 +1,4 @@
+name = "wrangler-dev-api-app"
+compatibility_date = "2022-03-31"
+
+main = "src/index.js"
```

Comment:
Given that you are passing the path to the entry-point in the call to `unstable_dev()` then this is not strictly needed here, right?

Is even the `name` property needed, too?

It might be nice to strip this example down to the bare bones of what is needed to do the API call.

(Alternatively, if `unstable_dev()` is reading this config file anyway, then perhaps we shouldn't be bothering to pass the path to the entry-point as an arg to that call?)